### ISSUE TITLE
OpenVPN release name

### ISSUE THREAD

**Issue by jza34** [dgalbraith/chocolatey-packages#511]
Hi, why do you name the current OpenVPN package 2.6.6.001 and not 2.6.6.1 ?
It creates undesirable side effect. Example if you list with choco you get v2.6.6.1 but if you upgrade with choco you get 2.6.6.001.
As we use Puppet with Chocolatey to manage software we always get a corrective upgrade from 2.6.6.1 to 2.6.6.001... which makes non sense.
Any reason ?
As I far as I read it on OpenVPN website, new releases always use minor upgrade enumerations: 2.6.1, 2.6.2, 2.6.4, etc. and never 2.6.5.1...

Thanks

**Comment by dgalbraith** [comment]
The minor versioning is actually driven by the versioning scheme applied to OpenVPN - the minor part of the version forms part of the underlying filename and is incremented.

```
$file32 = Join-Path $toolsDir 'OpenVPN-2.6.6-I001-x86.msi'
$file64 = Join-Path $toolsDir 'OpenVPN-2.6.6-I001-amd64.msi'
```

![image](https://github.com/dgalbraith/chocolatey-packages/assets/1270002/23b5d9f8-8095-4392-a15b-78a7f72045c5)

You are correct re the major naming convention users - the major part of the version OpenVPN-**2.6.6**-I001-amd64.msi represents the OpenVPN tag from which the version has been derived.  The 1st release for a particular tag would be listed as say ```2.7.0-I001``` with the ```2.7.0``` remaining constant across any future builds.

The minor portion is used to reflect updates due to changes made to bundled artifacts/binaries that are not part of the OpenVPN source code, say OpenSSL.  If a new version of OpenSSL were to be released an updated OpenVPN would be pushed and the minor portion incremented, say to ```2.7.0-I002``` to reflect this.

There is a discussion on this with one of the OpenVPN contributors in #452.

**Comment by dgalbraith** [comment]
I understand the existing behavior is not desirable, but it does feel to me that the versioning scheme is correct.

I would like to explore this a little with the Chocolatey devs to get a few from them - I don't see why the minor part of the version is having leading zeroes truncated.

That view will likely illuminate a path forward - either they agree and address the leading 0 truncation that is taking place in ```chocolatey list``` or I potentially look to move towards stripping the leading zeros from the minor version part eg. ```2.6.6.005``` would become ```2.6.6.5```

**Comment by dgalbraith** [comment]
@jza34 this should now be addressed.  The [standardisation](https://docs.chocolatey.org/en-us/choco/features/version-number-normalization) on the Chocolatey semantic versioning changed with version 2.0.0 and this has now been incorporated into the build and validation pipelines.

As of `OpenVPN 2.6.10.002` the leading zeros on version components will be truncated by Chocolatey on the `.nupkg` files.  `OpenVpn 2.6.10.002` is available as the Chocolatey package version [2.6.10.2](https://community.chocolatey.org/packages/openvpn/2.6.10.2).
